common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -236,10 +236,13 @@ object ContentWidths {
 
   object GalleryMedia {
     val inline = WidthsByBreakpoint(
-      mobile          = Some(445.px),
-      mobileLandscape = Some(610.px),
-      phablet =         Some(620.px),
-      tablet =          Some(700.px)) // desktop, leftCol, and wide are also 700px
+      mobile          = Some(480.px),
+      mobileLandscape = Some(660.px),
+      phablet =         Some(700.px),
+      tablet =          Some(700.px), // TODO: Change to 480 when new galleries is merged
+      desktop =         Some(720.px),
+      leftCol =         Some(880.px),
+      wide =            Some(1010.px))
```

Comment:
> BTW, I wonder how this works for portrait images that are now shown fluidly at the viewport height...

It looks pretty bad on a portrait device because the image is downloaded by width rather than height. We can update the value in our `sizes` attribute to fix this, so it downloads by the max of height/width, but it's a bit fiddly: codepen.io/tigt/post/when-responsive-images-get-ugly#height-and-width-constrained-srcset. I got it to work on http://samefourchords.com/ though :-)